src/conditional-flow/numeric-conditions.js

```diff
@@ -5,20 +5,35 @@ function isArrayEmpty (array) {
 
   // TODO: write code in this function body to pass the tests
 
+  if (array.length === 0){
+    return true
+  }
+  else {
+    return false
+  } 
 }
 
 // This function should return true if num1 is greater than num2, false otherwise
 function isGreaterThan (num1, num2) {
 
   // TODO: write code in this function body to pass the tests
 
+  if (num1 > num2){
+    return true
+  }
+  else {
+    return false
+  }
+
 }
 
 // This function should return the lowest number in the passed array
 function findLowest (nums) {
 
   // TODO: write code in this function body to pass the tests
-
+  lowestNum = Math.min(...nums);
```

Comment:
Idea here is to use your own implementation.

Have a look at Carlo and Shahzad's versions - which may be the same